loopFibonacci.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
```

Comment:
You should comment your code! Your code entered a loop that wouldn't really exit and then just output 0s.